utils/IrodsConnector.py

```diff
@@ -217,6 +213,14 @@ def password(self):
         """
         self._password = ''
 
+    def get_uid(self):
+        try:
+            uid = os.getuid()
+        except AttributeError:
+            # Spoof UID for Non-POSIX
+            uid = sum((ord(char) for char in os.getlogin()))
+        return uid
+
```

Comment:
Use of the `uid` is no longer necessary when `python_irodsclient==1.1.6` is used.  The PR (#67) for that is pending review.  This dependency can even be updated here for consistency if desired for this change.

Reply:
When [67](https://github.com/rijksuniversiteit-groningen/iBridges/pull/67) is approved and merge, we can merge develop back into this branch and fix it, do you agree?

Reply:
Yes.  Only one line remains in conflict.

Reply:
I merged the changes into this branch, there where a few small things but it works now.  And its really nice to login without having to retype the password every time.

Reply:
Indeed!